### ISSUE TITLE
vgg-m-2048 without BatchNorm model

### ISSUE THREAD

**Issue by noUmbrella** [albanie/pytorch-mcn#6]
It seems that, the magenet_mcn_vgg_m in your pre-convert model is the vgg-m with BN. Now, the pytorch supply the LRN, could you please supply the pre-convert model of vgg-m-2048 without BatchNorm but with LRN?

**Comment by Aakashagr** [comment]
Hi noUmbrella, 
Did you find out the solution to this?

@albanie, I am trying to convert a model with architecture similar to Alexnet shown in matconvnet website "[imagenet-caffe-alex](http://www.vlfeat.org/matconvnet/models/imagenet-caffe-alex.mat)" (after converting it into dagnnn). It contains a LRN layer which is not recognized by importer.py. Hence I am unable to convert it into pytorch format. Can you please help me with this? Thanks
